modules/BuildTools.Tasks/CreateSourceLink.cs

```diff
@@ -33,24 +35,24 @@ public class CreateSourceLink : Task
 
         public override bool Execute()
         {
-            var rootDirectory = Path.Combine(Path.GetFullPath(RootDirectory), "*");
-
-            var escapedPath = JsonEscapePath(rootDirectory);
+            var badPathSeparator = Path.DirectorySeparatorChar == '/' ? '\\' : '/';
+            var rootDirectory = Path.Combine(RootDirectory.Replace(badPathSeparator, Path.DirectorySeparatorChar), "*");
```

Comment:
Is this necessary? Won't this flip ```/``` to ```\``` when building on Windows CI (since the root is ```/_/```)?

Reply:
I thought this was a requirement for making source-link work....matching the type of slashes based on the current operating system. Does using PathMap mean we don't need this?

Reply:
Ideally we want the builds on Windows and Linux to produce the same paths embedded in PDBs and DLLs. That's why we use ```/_/``` for the deterministic root on both Windows and Linux.
If building locally, the RootDirectory is gonna be RepositoryRoot and that should be normalized already.

Reply:
Yes, when PathMap is used the compiler normalizes the separators.

Reply:
Also, you don't want to use ```Path.Combine``` as that uses platform-specific separator. Instead, I'd just assert that the path already ends with a separator (```/``` or ```\```).

Reply:
🙌 awesome.